src/test/scala/com/databricks/spark/redshift/RedshiftSourceSuite.scala

```diff
@@ -224,11 +217,7 @@ class RedshiftSourceSuite
             LessThanOrEqual("testInt", 43))
     val rdd = relation.asInstanceOf[PrunedFilteredScan].buildScan(Array("testByte", "testBool"), filters)
 
-    // We should now only have one matching row, with two columns
```

Comment:
Actually, I don't know that this test necessarily makes sense as written: it seems like we want our unit test to simply assert that the filter gets pushed down into Redshift. The end-to-end test to check whether we properly translate filters into `WHERE` clauses probably needs to run against a real database, so we can test the returned rows as part of that test.

For this test, I think that we should just write assertions over the exact queries passed to the mock database connection. I'm going to refactor this suite to do that.

Before I do that, though, a lot of the minor style and formatting issues in these files are bothering me, so I'm going to submit a separate style cleanup PR to implement Scalastyle checks so we never have to deal with this issue again.